test/manual/amp-consent/amp-consent-cmp-overlay.html

```diff
@@ -357,6 +361,11 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
   <footer>
     <div class="brand-logo">PublisherLogo</div>
   </footer>
+  <script>
+    setTimeout(() => {
```

Comment:
Presumably to avoid race condition...but can't find any documentation on this method; instead it looks like we should always use this: `    (self.AMP = self.AMP || []).push(function(AMP) {
      AMP.toggleExperiment('amp-consent-restrict-fullscreen', true);
    });`